pages/council.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div>
+  <div style="height: 100%;">
```

Comment:
Is this the main change? I'm not seeing it affecting anything on my screen but wondering if I perhaps don't have the right resolution to show the issue. Are you able to give a bit of context for this if this is the crux of the change? Thank you

Reply:
This is how it was looking before on mobile devices before adding this. It didn't take up the full space available.
![image](https://github.com/uoe-dlam/witches/assets/71814229/9b084763-4c6c-4dde-8827-2ad319079251)
This how it now looks on mobile devices, taking up the full screen.
![image](https://github.com/uoe-dlam/witches/assets/71814229/52e4b2c2-464b-4f2c-a638-7322eacdf1d8)